### PR TITLE
Add global yarn

### PR DESCRIPTION
**Pull request by lillianzhang331**
Handling a vendored yarn by having the buildpack create an alias

### CODE REVIEW

**Comment by jamuna63623** [comment]
Error: The buildpack heroku/nodejs is already set on your app.

**Comment by danielleadams** [comment]
@jamuna63623 This is a pull request for a feature not in the buildpack yet - did you mean to open an issue?

File: README.md [hunk 1]
```diff
@@ -107,6 +107,14 @@ The tests are run via the vendored
 [shunit2](https://github.com/kward/shunit2)
 test framework.
 
+### Debugging
+
+To show the logged build outputs:
```
Comment (danielleadams) [inline README.md:112]: Thanks for adding this! Do you mind adding a little more description about this? Maybe where exactly it should go, or an example snippet within the test?

File: README.md [hunk 2]
```diff
@@ -107,6 +107,35 @@ The tests are run via the vendored
...
+  assertFileContains "test log file" "$log_file"
+}
+```
+Running the test above would produce:
+```
+testtest
+this is the log file
+test log file
+```
+
```
Comment (danielleadams) [inline README.md:138]: This looks great! I would just leave one more comment that mentions that the test output already writes to `$STD_OUT` at the bottom.

**Pull request merged**